e2-studio-star-rx72n-firmware/tests/test_rx_motor.c

```
@@ -678,30 +746,30 @@ void test_motor_invert_pwm_reverse_becomes_forward(void)
+                           0.0F,
                            mock_gptw_get_duty(k_gptw_channel_0, k_gptw_output_b));
 }
 
 void test_motor_stop_brake_sets_duty_zero(void)
 {
   TEST_ASSERT_EQUAL(k_rx_ok, rx_motor_init(&s_motor, &s_config));
-  TEST_ASSERT_EQUAL(k_rx_ok, rx_motor_set_duty(&s_motor, 75.0f));
+  TEST_ASSERT_EQUAL(k_rx_ok, rx_motor_set_duty(&s_motor, 75.0F));
   TEST_ASSERT_EQUAL(k_rx_ok, rx_motor_stop(&s_motor, true));
 
   float duty;
```

Clang (14.0.6)

[warning] variable 'duty' is not initialized

(cppcoreguidelines-init-variables)